go.mod

```diff
@@ -15,10 +15,10 @@ require (
 	github.com/urfave/cli/v2 v2.2.0
```

Comment:
why upgrade the dependencies with this PR ? 🤔

Reply:
it's always done with go mod tidy

Reply:
go mod tidy is always called with `make check